styles.css

```diff
@@ -552,32 +530,25 @@ p {
 
```

Comment:
Great work handling responsive design with multiple breakpoints!! You've really nicely covered a wide range of screen sizes here.

I would even consider adding a few more breakpoints to smooth out the transitions as you grow and shrink your page. Below you'll find a screenshot with one template for breakpoints! These are used in [Bootstrap](https://getbootstrap.com/docs/5.0/layout/breakpoints/#available-breakpoints), which is a styling package that might be cool to explore in Mod 4 as stretch tech. In addition to dragging the responsive window width, I also like using the dev tools dropdown for common devices to get a sense for UX on an iPad vs. iPhone, when their device is in landscape vs. portrait, etc.
![Screen Shot 2021-04-11 at 6 21 23 PM](https://user-images.githubusercontent.com/62263439/114326586-bab32d80-9af2-11eb-99bf-e1a3dc3302e8.png)

<img width="818" alt="Screen Shot 2021-04-11 at 6 17 43 PM" src="https://user-images.githubusercontent.com/62263439/114326477-395b9b00-9af2-11eb-9a2d-7380025ee5b9.png">